src/components/SubscribeNewsletter/subscribe-newsletter.tsx

```diff
@@ -0,0 +1,40 @@
+import { SquiggleLeft, SquiggleRight } from '@/assets/svg'
+import Button from '../Button/button'
+
+function SubscribeNewsLetter() {
+  return (
+    <div className="w-full relative bg-ibtc-light-yellow flex flex-col items-center justify-center min-h-[26rem] gap-[2rem]">
+      <div className="text-center space-y-[1rem]">
+        <h2 className="text-[2rem] lg:text-[3.438rem] font-[500] m-0 p-0">
```

Comment:
Also after adding the className "_header" give it the color.
In this case, the color is `#000000` instead of `text-ibtc-black`
![Screenshot (50)](https://user-images.githubusercontent.com/51183064/231301574-21f18c0e-50fa-4e5b-8e4b-ffe019c69e9c.png)